### DISCUSSION TITLE
Update wiki article https://github.com/Daveiano/weewx-wdc/wiki/Support-for-weewx-DWD

### DISCUSSION THREAD

**Discussion by roe-dl** [Daveiano/weewx-wdc#283] [General]
Within the last few months there were some major updates of [weewx-DWD](https://github.com/roe-dl/weewx-DWD). Some functions described in [Support for weewx-DWD](https://github.com/Daveiano/weewx-wdc/wiki/Support-for-weewx-DWD) wiki page are deprecated now and replaced by an easier way to set up and configure. Furthermore there are new functions and support for more weather services, for example KNMI, OpenWeather, and EUMETSAT.

**Comment by Daveiano** [comment]
Thanks for the remider! Will have a look at the projects readme and update the wdc wiki as soon as possible!

**Comment by Daveiano** [comment]
@roe-dl I had a read over the readme and some wiki pages and I got a few questions (The project got really big - I am feeling a bit overwhelmed to be honest of all the different features):

- Both, wget-dwd and dwd-cap-warnings are marked deprecated - but where are the replacements? Also here https://github.com/roe-dl/weewx-DWD/tree/master?tab=readme-ov-file#installation under manual installation the files are still mentioned (the wiki also mentions these scripts https://github.com/roe-dl/weewx-DWD/wiki/Deutscher-Wetterdienst#aktuelle-me%C3%9Fwerte-von-dwd-wetterstationen-als-werte-in-weewx)
- There are multiple providers/models described in https://github.com/roe-dl/weewx-DWD/tree/master?tab=readme-ov-file#weather-services-and-productsweather-models - any way to use multiple of them at the same time? 

**EDIT** I guess I found the replacements for the text forecasts of wget-dwd, it is described here https://github.com/roe-dl/weewx-DWD/wiki/Deutscher-Wetterdienst#text-wettervorhersage. And the images (bwk_bodendruck and SchilderLZ images I guess) are described here https://github.com/roe-dl/weewx-DWD/tree/master?tab=readme-ov-file#weather-services-and-productsweather-models at the very bottom.

Note to myself: Have a look at https://github.com/roe-dl/weewx-DWD/tree/master?tab=readme-ov-file#searchlist-extension-presentweather and probably add default support for it in the skin.

**Comment by roe-dl** [comment]
In general the target of that extension is "weather services". As there are many weather services, there are many options. Sorry for confusing you.

* `dwd-warnings` (without `cap`) is deprecated. `dwd-cap-warnings` should be used.
* `wget-dwd` is not necessary any more. Downloads are done by setting up a `[[download]]` section in `weewx.conf`. This is more flexible and easier to be configured.
* I moved to write a separate wiki page for each of the weather services. It is too much for a single readme file. The home page of the wiki contains a list of the weather services along with links to the specific pages.
* Yes, you can use them in parallel.
* `SchilderLZ` etc. is described in the wiki article [Warnstatuskarten](https://github.com/roe-dl/weewx-DWD/wiki/Warnstatuskarten).
* And I see, `bwk-bodendruck` is missing in the wiki.

**Reply by roe-dl** [reply]
`bwk-bodendruck` is now included in the wiki: [Bodenwetterkarte](https://github.com/roe-dl/weewx-DWD/wiki/Deutscher-Wetterdienst#bodenwetterkarte)

**Reply by Daveiano** [reply]
Great, thank you! Will have a look and update the wiki soon!
